.github/actions-rs/grcov.yml

```diff
@@ -0,0 +1,13 @@
+branch: true
```

Comment:
As a general note on github actions, it would be good to make sure we didn't substantially alter the config and will report very different cov numbers after merging. I doesn't seem so but I didn't look into the type-script code for this action.

Reply:
Just noticed this (too late!)

I'd love to give another shout out to Tarpaulin. It's pretty much the go-to coverage tool for the Rust community, IMO:

https://github.com/actions-rs/tarpaulin

I've personally experienced a lot of false negatives trying grcov in the past. I don't know if they've gotten any better, but if you see things you think should be covered under grcov which it reports aren't... maybe check out tarpaulin.

Here's one of my example configs, FWIW:

https://github.com/RustCrypto/AEADs/blob/master/.github/workflows/workspace.yml#L41